sources/web/datalab/server.ts

```diff
@@ -185,6 +186,21 @@ function handleRequest(request: http.ServerRequest,
     return;
   }
 
+  if (path.indexOf('/_killvm') == 0) {
+    if ('POST' != request.method) {
```

Comment:
I think it would be cleaner if this whole block was moved to a helper method called something like "stopHandler"